Formula/cryfs.rb

```diff
@@ -0,0 +1,30 @@
+require "open3"
+
+class Cryfs < Formula
+  desc "Encrypts your files, so you can safely store them in Dropbox, iCloud, ..."
+  homepage "https://www.cryfs.org"
+  url "https://github.com/cryfs/cryfs.git", :tag => "0.9.7"
+  head "https://github.com/cryfs/cryfs.git", :branch => "develop"
+
+  depends_on "cmake" => :build
+  depends_on :python => :build
+  depends_on "openssl"
+  depends_on "boost"
+  depends_on "cryptopp"
+  depends_on :osxfuse
+
+  needs :cxx11
+
+  def install
+    system "cmake", ".", "-DBUILD_TESTING=off", *std_cmake_args
+    system "make", "install", "prefix=#{prefix}"
+  end
+
+  test do
+    # Mount file system
+    Open3.popen3({ "CRYFS_FRONTEND" => "noninteractive" }, "#{bin}/cryfs") do |stdin, stdout, _|
```

Comment:
changed it. It's testing that the executable was installed correctly and can be run. That's all we can do because we can't actually mount something as discussed above.

Since we don't actually mount something, the exit status will be 1, which I added as a parameter to shell_output.